packages/bridge/smart-contract-templates/icon/icon_hello/icon_hello.py.hbs

```diff
@@ -22,12 +22,14 @@ class hello(IconScoreBase):
         return "Hi"
 
     {{#each tables}}
-    @external
+    ################################################
+    # "{{name}}" supported functions
+    ################################################
+
+    @external(readonly=True)
```

Comment:
This update belongs to the bridge scope but there is no commit which mentioned it.

BTW, should we continue to merge this into the main branch? I saw it really hurt the coverage.

Reply:
I think you mistaken it, the coverage is slightly changed (0.08%) compare to previous commit. Btw, I'll update to fill the gap.

Reply:
I forgot that the PR https://github.com/lecle/aloxide/pull/110 add a lot of test cases, I'll create another PR for the missing cases.